tests/bbsplit.nf.test

```diff
@@ -19,7 +19,8 @@ nextflow_pipeline {
                 modules_testdata_base_path: 'https://raw.githubusercontent.com/nf-core/test-datasets/modules/data/',
                 igenomes_base: 'https://raw.githubusercontent.com/nf-core/test-datasets/modules/data/',
                 genome: 'testdata.nf-core.sarek'
-            ]
+            ],
+            ignoreFiles: '{multiqc/multiqc_data/mosdepth-coverage-per-contig-single.txt,multiqc/multiqc_data/mosdepth-cumcoverage-dist-id.txt,multiqc/multiqc_data/mosdepth_perchrom.txt,multiqc/multiqc_data/samtools*.txt,preprocessing/**/*.cram,reports/mosdepth/*/*}'
```

Comment:
:chefskis: although I hate tthat we have so many files with instable sums. in round two maybe we can get better at checking the content

Reply:
yeah, but that's just bbsplit